src/terra_validator.js

```diff
@@ -0,0 +1,8 @@
+var bech32 = require('./crypto/bech32');
+
+module.exports = {
+    isValidAddress: function (address, currency, opts) {
+      const decoded = bech32.decode(address, bech32.encodings.BECH32);
+      return (decoded) !== null && decoded.hrp === 'terra';
```

Comment:
kalo aku liat dari sini https://github.com/pintu-crypto/wallet-address-validator/blob/master/src/crypto/bech32.js#L119
hrp itu kelihatannya panjang, jd gimana kalau dibuat gini?
```suggestion
      return (decoded) !== null && decoded.hrp.startsWith('terra');
```